### ISSUE TITLE
message: [generated]:8: warning: invalid argument for command '\iline'

### ISSUE THREAD

**Issue by aotto1968** [doxygen/doxygen#10932]
**Describe the bug**
I get a "iline" message together with @copybrief into a @name section

**Screenshots**
![image](https://github.com/doxygen/doxygen/assets/27886664/5f63fde4-aba4-4649-a75d-6e7d0fcc0a81)

**To Reproduce**
use the attached example

**Expected behavior**
no "iline" message

**Version**
1.10.0

**Additional context**
[doxygen_bug_iline.tar.gz](https://github.com/user-attachments/files/15775665/doxygen_bug_iline.tar.gz)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #10933

**Comment by aotto1968** [comment]
I have no clue how to use the "pull-request" → it is possible to provide a plain git command?

**Comment by albert-github** [comment]
I don't have a direct git command either, when I want to try a patch I always use one of the following:
- check the differences in the patch and apply them manually
- clone the repository (in this case: https://github.com/albert-github/doxygen.git), select the appropriate branch and build.
- As I normally get an email with the changes in the original patch (unfortunately not when a patch is updated :-( ) I can apply one of the things mentioned there. In this case the email looked like:
  ```
  For generated files also a line number -1 is possible with the iline command.
  You can view, comment on, or merge this pull request online at:

    https://github.com/doxygen/doxygen/pull/10933
  Commit Summary

      [6be1731](https://github.com/doxygen/doxygen/pull/10933/commits/6be1731bca0f7ab3bba01a4d6b006ea80957a008) issue #10932 message: [generated]:8: warning: invalid argument for command '\iline'

  File Changes

  ([1 file](https://github.com/doxygen/doxygen/pull/10933/files))

      M [src/doctokenizer.l](https://github.com/doxygen/doxygen/pull/10933/files#diff-d6db1718954414bc29d3154ced1a0a3fbf6c3867533a3f3e507131421a2a8cf5) (2)

  Patch Links:

      https://github.com/doxygen/doxygen/pull/10933.patch
      https://github.com/doxygen/doxygen/pull/10933.diff
```

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.12.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
